FEXCore/Source/Interface/Core/JIT/JIT.cpp

```diff
@@ -755,6 +753,10 @@ CPUBackend::CompiledCode Arm64JITCore::CompileCode(uint64_t Entry, uint64_t Size
     CTX->ClearCodeCache(ThreadState);
   }
 
+  // TODO: Carefully aligned to ensure InstCountCI produces consistent results. Should be moved to CodeSizeValidation.
```

Comment:
Is this TODO planned to be fixed in this PR or later?

Reply:
I wasn't entirely sure if it's a good way to address the InstCountCI issue to begin with. If it is, I'll add a proper interface as part of this PR.